packages/discord.js/src/managers/RoleManager.js

```diff
@@ -109,12 +109,23 @@ class RoleManager extends CachedManager {
    * @returns {?Snowflake}
    */
 
+  /**
+   * @typedef {Object} RoleColors
+   * @property {number} primaryColor The primary color of the role
+   * @property {?number} secondaryColor The secondary color of the role.
+   * This will make the role a gradient between the other provided colors
+   * @property {?number} tertiaryColor The tertiary color of the role.
+   * When sending `tertiaryColor` the API enforces the role color to be a holographic style with values of `primaryColor = 11127295`, `secondaryColor = 16759788`, and `tertiaryColor = 16761760`
+   */
+
   /**
    * Options used to create a new role.
    *
    * @typedef {Object} RoleCreateOptions
    * @property {string} [name] The name of the new role
-   * @property {ColorResolvable} [color] The data to create the role with
+   * @property {ColorResolvable} [color] The color to create the role with.
+   * This will still be returned by the API, but using the `colors` field is recommended when doing requests
```

Comment:
I know this is the wording in the docs, but to me it reads like this should get a `deprecated` treatment or (since this targets main) simply be removed since djs shouldn't maintain both fields in the future.

Reply:
> since djs shouldn't maintain both fields in the future
If both are supported at the API level we'll support both

Reply:
Message#interaction is also "supported" at the API level, yet we removed it in main and deprecated in v14, because it's deprecated on API level...

Reply:
Color isn't deprecated tho, is it?

Reply:
It is not marked as deprecated in the documentation.